app/views/partials/_comments.html.erb

```diff
@@ -1,4 +1,15 @@
-<li class="border border-1 p-1 rounded mb-1">
-  <span><%= comment.author.name %></span>:
-  <span><%= comment.text %></span>
+<li class="border border-1 p-1 rounded mb-1 d-flex justify-content-between">
+  <div>
+    <span><%= comment.author.name %></span>:
+    <span><%= comment.text %></span>
+  </div>
+  <div>
+    <% if can? :destroy, comment %>
+      <%= link_to 'delete', [current_user, @post, comment], data: {
+        turbo_method: :delete,
+        turbo_confirm: 'Are you sure!'
+        }, class: 'btn btn-danger btn-sm' 
+      %>
+    <% end %>
+  </div>
```

Comment:
- [ ] Kindly fix this error which is due to the wrong path put in the link_to tag
<img width="1081" alt="image" src="https://github.com/zeff96/blog_app/assets/65068771/a834aa26-20c7-41d9-b060-51a6542f669c">